### PR TITLE
Update APIs Booking with demo suggestions

### PR DESCRIPTION
**Pull request by aquena-code**

### CODE REVIEW

File: BOOKING_SERVICE/main.py [hunk 1]
```diff
@@ -21,136 +21,132 @@
 
 
 # Create a Resource for Booking
-@app.route('/CreateResource', methods=['POST'])
+@app.route('/resource', methods=['POST'])
 def create_resource():
     resource = Resource()
     result = resource.resource_create(request.json)
     return jsonify(result)
```
Comment (FreddyClaros) [inline BOOKING_SERVICE/main.py:29]: here add one more space please

File: BOOKING_SERVICE/main.py [hunk 2]
```diff
@@ -21,136 +21,132 @@
...
 
 
 # Update a Person by Id of the Database of Booking
-@app.route('/UpdatePerson/<string:id_person>', methods=['PUT'])
+@app.route('/person/<string:id_person>', methods=['PUT'])
 def update_person(id_person):
     person = Person()
     result = person.person_update(id_person, request.json)
     return jsonify(result)
```
Comment (FreddyClaros) [inline BOOKING_SERVICE/main.py:108]: one more space please between functions please

File: BOOKING_SERVICE/main.py [hunk 3]
```diff
@@ -21,136 +21,132 @@
...
 
-
-@app.route('/DeletePerson/<string:id_person>', methods=['DELETE'])
+# Delete a Person by Id of the Database of Booking
+@app.route('/person/<string:id_person>', methods=['DELETE'])
```
Comment (FreddyClaros) [inline BOOKING_SERVICE/main.py:110]: Are you sure that only /person/<string: .... > will be enough for know that its a booking process?  If you do, don't worry and its ok

**Review by Amilcar-B — APPROVED** [review]

**Review by FreddyClaros — APPROVED** [review]
Now all seems ok, good Job

**Pull request merged**